config/config/sc-config.yaml

```diff
@@ -899,6 +899,18 @@ opensearch:
     nodeSelector: {}
     tolerations: []
 
+  promIndexAlerts:
+    - pattern: kubernetes-default
+      alertSizeMB: 5500
+    - pattern: kubeaudit-default
+      alertSizeMB: 5500
+    - pattern: other-default
+      alertSizeMB: 400
+    - pattern: security-auditlog
+      alertSizeMB: 16
+    - pattern: authlog-default
+      alertSizeMB: 0.2
```

Comment:
Any specific reason why 200KB chosen when the retention size is 1GB?
Feels like we could choose a % of the retention size instead of specifying MB here. So if you choose a new retention size the alert follows.

Reply:
I agree, but I do not think this index will every reach 1GB in size (at least for the env I checked) so maybe we should re-evaluate the retention size for security-auditlo and authlog-default
do you want to do that as part of this PR?

Reply:
Yes maybe worth to look at the retention size. No I would say create an new issue for it.